### PR TITLE
refactor: patch params editor

### PR DESCRIPTION
**Pull request by cpvalente**
Small fixes to the params editor feature

### CODE REVIEW

**Comment by cpvalente** [comment]
Hi @asharonbaltazar ,

Would you mind sanity-checking here?
I had to make a few changes to the properties and styling.
Also made changes to the feature UX (keep form open on submit and add a clear button)

Please ignore the lower thirds.
It is a very painful component that is fragile due to the framer motion animations. I will cleanup someday

File: apps/client/src/common/components/edit-form-drawer/EditFormDrawer.tsx [hunk 1]
```diff
@@ -82,6 +85,9 @@ export default function EditFormDrawer({ paramFields }: EditFormDrawerProps) {
         </DrawerBody>
 
         <DrawerFooter className={style.drawerFooter}>
+          <Button variant='ontime-ghosted' onClick={clearParams} type="reset">
```
Comment (asharonbaltazar) [inline apps/client/src/common/components/edit-form-drawer/EditFormDrawer.tsx:88]: super tiny nit: Prettier is complaining about this

**Comment by asharonbaltazar** [comment]
Also, I forgot to mention, that `<input type='number' />` doesn't actually work 🤷‍♂️. You can still input letters and whatnot.

**Review by asharonbaltazar — APPROVED** [review]
Otherwise, the changes that pertain to `EditFormDrawer` are great! 

(maybe it's worth renaming these related components to `CustomiseFormDrawer` for better clarity? Just a thought)

File: apps/client/src/common/components/navigation-menu/NavigationMenu.module.scss [hunk 2]
```diff
@@ -46,7 +47,6 @@ $button-size: 48px;
 
 .navButton {
   @extend .button;
-  z-index: 2;
```
[the remark this answers is not in the document]
Comment (cpvalente) [reply apps/client/src/common/components/navigation-menu/NavigationMenu.module.scss:49]: nice, good catch. I did wonder "what a strange choice of design" and moved on 😅 
let the z-index dance begin!

**Review by asharonbaltazar — APPROVED** [review]
Great work!

**Pull request merged**